samples/chat/csharp-ooproc/ChatBot.cs

```diff
@@ -40,7 +40,11 @@ public class CreateRequest
         return new CreateChatBotOutput
         {
             HttpResponse = new ObjectResult(responseJson) { StatusCode = 201 },
-            ChatBotCreateRequest = new AssistantCreateRequest(chatId, createRequestBody.Instructions),
+            ChatBotCreateRequest = new AssistantCreateRequest(chatId, createRequestBody.Instructions)
+            {
+                ChatStorageConnectionSection = "AzureWebJobsStorage",
+                CollectionName = "SampleChatState"
```

Comment:
this is where users can pass the environment variables.
to pass environment variables, use %Var_Name%

for example
ChatStorageConnectionSection = "%ChatStorage%"
or 
ChatStorageConnectionSection = "%ChatStorage_tableServiceUri%"